packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -1437,6 +1476,17 @@ abstract class NetworkImage extends ImageProvider<NetworkImage> {
   /// When running Flutter on the web, headers are not used.
   Map<String, String>? get headers;
 
+  /// On the Web platform, specifies when the image is loaded as a
+  /// [WebImageInfo], which causes [Image.network] to display the image in an
+  /// HTML <img> tag in a platform view.
+  ///
+  /// See [Image.network] for more explanation.
+  ///
+  /// Defaults to [WebImgElementStrategy.never].
+  ///
+  /// Has no effect on other platforms, which always fetch bytes.
+  WebImgElementStrategy get useImgElement;
```

Comment:
`useImageElement` sounds like a boolean property. I'd call it `*Strategy`, e.g. `sameOriginStrategy` (especially if we rename the enum to `SameOriginStrategy`.